src/OrganizationIntegration/OrganizationIntegrationCreate/OrganizationIntegrationCreate.js

```diff
@@ -39,7 +43,7 @@ const buildInitialValues = (organization, withMigration) => {
           vendorEdiType: edi.vendorEdiType || EDI_CODE_TYPES[0].value,
           libEdiCode: edi.libEdiCode,
           libEdiType: edi.libEdiType || EDI_CODE_TYPES[0].value,
-          ediNamingConvention: edi.ediNamingConvention,
+          ediNamingConvention: edi.ediNamingConvention || getDefaultEdiNamingConvention(),
```

Comment:
feel free to remove edi.ediNamingConvention, it was done to migrate previous edi configs, now it doesn't make sense with real default

Reply:
updated, thanks